src/index.html

```diff
@@ -7,6 +7,38 @@
     <link rel="stylesheet" href="./styles/main.scss">
   </head>
   <body>
-    <h1>Calendar</h1>
+    <div class="calendar calendar--month-length_30 calendar--start-day_thu">
```

Comment:
```suggestion
    <div class="calendar calendar--month-length--30 calendar--start-day--thu">
```
![image](https://user-images.githubusercontent.com/96409563/226164722-a971e286-81de-417c-92cd-9b98c3f8478d.png)